src/main/java/com/avispl/symphony/dal/avdevices/power/aten/pe4104g/AtenPDUCommunicator.java

```diff
@@ -262,7 +265,9 @@ private void retrieveMonitoring() throws Exception {
 					case OUTLET_STATUS_2:
 					case OUTLET_STATUS_3:
 					case OUTLET_STATUS_4:
-						cacheKeyAndValue.put(command.getName(), getDefaultValueOrNone(response.split("\r\n")[1].trim()));
+						if (response.split("\r\n").length > 1) {
```

Comment:
can you please make response.split("\r\n") a variable, so it doesn't get executed twice?